### DISCUSSION TITLE
Switching The UEFI bootloader

### DISCUSSION THREAD

**Discussion by behdanisohrab** [parchlinux/.github#11] [Polls]
Greetings to the Parch distribution community.

Due to the problems with grub in some devices (not being recognized in UEFI boot and manual recognition of this boot manager), the pitcher team intends to replace this boot manager with systemd-boot to better support the devices.

**Comment by ropinout** [comment]
dude Wouldn't that make us more dependent on the systemd?
Do you have any plans to change the init?

**Reply by behdanisohrab** [reply]
not specifically.
this boot problem occurs mostly with grub and this brands:
dell, acer, asus, apple

we have plans to change the init system and use s6 in the future but its not happening anytime soon because of some software and repo problems.
